tests/test_lanes.py

```diff
@@ -341,7 +342,7 @@ def test_initialization(self):
         self._db.commit()
 
         assert_raises(
-            ValueError, RelatedBooksLane, self._default_library, self.work, ""
+            CannotLoadConfiguration, RelatedBooksLane, self._default_library, self.work, ""
```

Comment:
I think this is causing your final failing test. RelatedBooksLane.__init__ in api/lanes.py is raising its own ValueError, so if you change this back to ValueError the tests should all pass.